pkg/blocktx/client.go

```diff
@@ -2,18 +2,18 @@ package blocktx
 
 import (
 	"context"
+	grpcopts "github.com/bitcoin-sv/arc/internal/grpc_opts"
```

Comment:
No alias needed as the rule has been excluded

Reply:
Removed